src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -217,15 +234,25 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
             },
 
             innerText: {
-                condition: el => typeof el.tagName === 'string' && domUtils.isScriptElement(el) &&
-                                 typeof el.innerText === 'string',
+                condition: el => typeof el.tagName === 'string' && typeof el.innerText === 'string' &&
+                                 (domUtils.isScriptElement(el) || domUtils.isStyleElement(el)),
```

Comment:
`typeof el.tagName === 'string' && typeof el.innerText === 'string' ` 

is it necessary ?